docs/tenant_crd.adoc

```diff
@@ -804,6 +808,18 @@ TenantSpec (`spec`) defines the configuration of a MinIO Tenant object. +
 |*Optional* + 
  Specify a secret that contains additional environment variable configurations to be used for the MinIO pools. The secret is expected to have a key named config.env containing all exported environment variables for MinIO+
 
+|*`initContainers`* __link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#container-v1-core[$$Container$$] array__ 
+|*Optional* + 
+ Add customs initContainers to StatefulSet
+
+|*`additionalVolumes`* __link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#volume-v1-core[$$Volume$$] array__ 
+|*Optional* + 
+ If provided, work statefulset will add this volumes. You should set the rules for the corresponding volume and VolumeMount. We will not test this rule, K8S will show the result.
+
+|*`additionalVolumeMounts`* __link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#volumemount-v1-core[$$VolumeMount$$] array__ 
+|*Optional* + 
+ If provided, work statefulset will add this volumemunts. You should set the rules for the corresponding volume and VolumeMount. We will not test this rule, K8S will show the result.
```

Comment:
```
If provided, statefulset will add these volume mounts. You should set the rules for the corresponding volumes and volume mounts. We will not test this rule,  k8s will show the result.
```

Reply:
applied the suggestion. Thanks @shtripat